hack/update-estimator-protobuf.sh

```diff
@@ -14,7 +14,7 @@ export PATH=$PATH:$GOPATH/bin
 GO111MODULE=on go install golang.org/x/tools/cmd/goimports
 GO111MODULE=on go install k8s.io/code-generator/cmd/go-to-protobuf
 GO111MODULE=on go install github.com/gogo/protobuf/protoc-gen-gogo
-GO111MODULE=on go install github.com/vektra/mockery/v2
+GO111MODULE=on go install github.com/vektra/mockery/v2@v2.10.0
```

Comment:
```suggestion
GO111MODULE=on go install github.com/vektra/mockery/v2
```
This change should be revert here, `go install` will install the binary according to `go.sum` where we manages the dependencies.

Reply:
I forgot to make a change.